examples/netweaver_simple_example/main.tf

```
@@ -41,4 +41,37 @@ module "gcp_netweaver" {
   boot_disk_type         = var.boot_disk_type
   disk_type              = var.disk_type
   startup_script         = var.startup_script
+  pd_kms_key             = google_kms_crypto_key.netweaver_simple.self_link
+}
+
+# Create a KMS key to use as customer managed encryption key for the instance
+# persistent disk. This is completely optional. If you do not need to manage
+# your own keys, just remove this section and remove also the pd_kms_key
+# parameter in the module declaration above.
+resource "google_kms_key_ring" "netweaver_simple" {
+  project  = var.project_id
+  name     = "netweaver-simple-${random_id.this.hex}"
+  location = var.region
+}
+
+resource "google_kms_crypto_key" "netweaver_simple" {
+  name     = "netweaver-simple-${random_id.this.hex}"
+  key_ring = google_kms_key_ring.netweaver_simple.self_link
+}
+
+data "google_project" "project" {
+  project_id = var.project_id
+}
+
+resource "google_kms_crypto_key_iam_binding" "netweaver_simple" {
```

In general, I suggest not using "binding" resources without a strong reason. They're often footguns.

```suggestion
resource "google_kms_crypto_key_iam_member" "netweaver_simple" {
```

---

I agree. Changed it to a member resource.